src/Core/Services/AuthService.cs

```diff
@@ -198,12 +198,40 @@ private async Task<bool> RequirePasswordChangeAsync(string email, string masterP
             return !await _policyService.EvaluateMasterPassword(strength.Value, masterPassword, _masterPasswordPolicy);
         }
 
-        public async Task<AuthResult> LogInPasswordlessAsync(string email, string accessCode, string authRequestId, byte[] decryptionKey, string userKeyCiphered, string localHashedPasswordCiphered)
+        public async Task<AuthResult> LogInPasswordlessAsync(string email, string accessCode, string authRequestId, byte[] decryptionKey, string masterKey, string masterKeyHash)
```

Comment:
Can we rename `masterKey` since it won't always be a Master Key? Something like `encryptedAuthRequestKey` would be better. Also putting some comments in this method would really help readability.